packages/bot-web-ui/src/components/dashboard/tutorial-tab/common/search-input/index.tsx

```diff
@@ -2,17 +2,39 @@ import { observer } from '@deriv/stores';
 import { localize } from '@deriv/translations';
 import { useDBotStore } from 'Stores/useDBotStore';
 import React from 'react';
+import { Analytics } from '@deriv/analytics';
+import debounce from 'lodash.debounce';
+import { DEBOUNCE_INTERVAL_TIME } from 'Constants/bot-contents';
```

Comment:
should this folder has name uppercased?

Reply:
actually we use constants as caps in our package 
<img width="964" alt="image" src="https://github.com/binary-com/deriv-app/assets/97010868/5e1c1338-c913-442c-9de3-4b7de9c54ddb">

Reply:
haha, that...suddenly. but okay, thanks for explanation